setup.py

```diff
@@ -851,20 +871,31 @@ def detect_modules(self):
                                ['/usr/kerberos/include'])
             if krb5_h:
                 ssl_incs += krb5_h
-        ssl_libs = find_library_file(self.compiler, 'ssl',lib_dirs,
+        if host_platform == 'vxworks':
```

Comment:
The code below is mostly duplicated. The only differences are few arguments. Wouldn't be easy to just set platform depending local variables for these arguments? The value for `libraries` also is used below.